vhdbuilder/packer/install-dependencies.sh

```diff
@@ -319,6 +323,7 @@ SWIFT_CNI_VERSIONS="
 for VNET_CNI_VERSION in $SWIFT_CNI_VERSIONS; do
     VNET_CNI_PLUGINS_URL="https://acs-mirror.azureedge.net/azure-cni/v${VNET_CNI_VERSION}/binaries/azure-vnet-cni-swift-linux-${CPU_ARCH}-v${VNET_CNI_VERSION}.tgz"
     downloadAzureCNI
+    unpackAzureCNI
```

Comment:
won't this overwrite the files every time? or do I need some sleep

Reply:
only thing changing is source, and dest is the same every time?

Reply:
hmm, no I think it's okay...will check logs tomorrow...

VNET_CNI_PLUGINS_URL -> last part becomes CNI_TGZ_TMP
CNI_TGZ_TMP without .tgz becomes CNI_DIR_TMP

$CNI_DOWNLOADS_DIR/$CNI_DIR_TMP is final unpack dir...

so as long as the file name/versions are unique for each version, it works?

Reply:
aaaand for my own sanity: from the runtime side --

https://github.com/Azure/AgentBaker/blob/39611fbce3331dae7e9c3598134c5e737c1d52ee/parts/linux/cloud-init/artifacts/cse_install.sh#L194-L195

https://github.com/Azure/AgentBaker/blob/39611fbce3331dae7e9c3598134c5e737c1d52ee/parts/linux/cloud-init/artifacts/cse_install.sh#L200-L202

still looking in $CNI_DOWNLOADS_DIR/$CNI_DIR_TMP, then mv to final location, and fall back to tarball, then fall back to download.

bravo for managing this